File: examples/java/java-booking-04_tell_dont_ask/src/main/java/tv/codely/booking/Customer.java
package tv.codely.booking;

public final class Customer {
    private final CustomerId   id;
    private final CustomerName name;
    private final EmailAddress emailAddress;

    public Customer(CustomerId id, CustomerName name, EmailAddress emailAddress) {
        this.id           = id;
        this.name         = name;
        this.emailAddress = emailAddress;
    }
}

File: examples/csharp/csharp-booking-02_introduce_parameter_object/src/Booking/Customer.cs
namespace CodelyTv.Booking
{
    public sealed class Customer
    {
        public CustomerId Id { get; }
        public CustomerName Name { get; }
        public EmailAddress EmailAddress { get; }

        public Customer(CustomerId id, CustomerName name, EmailAddress emailAddress)
        {
            this.Id = id;
            this.Name = name;
            this.EmailAddress = emailAddress;
        }
    }
}
